.circleci/config.yml

```diff
@@ -99,15 +99,19 @@ workflows:
   build-and-publish:
     jobs:
       - build:
+          context: maven-central-publish
           filters:
             tags:
               only: /.*/
       - muzzle:
+          context: maven-central-publish
           filters:
             tags:
               only: /.*/
       - release-bintray-publish:
```

Comment:
this job is no longer going to bintray so should be renamed

Reply:
done.